src/bp/ui-studio/src/web/views/FlowBuilder/nodeProps/SkillCallNode.jsx

```diff
@@ -10,7 +10,13 @@ const style = require('./style.scss')
 
 export default class SkillCallNodePropertiesPanel extends Component {
   renameNode = text => {
-    if (text && text !== this.props.node.name) {
+    if (text) {
+      let nodes = this.props.flow.nodes;
+      for (let i = 0; i < nodes.length; i++) {
+        if (text === nodes[i].name) {
+          return;
+        }
+      }
```

Comment:
Please make sure you have Prettier and TSLint enabled on VSCode. Coding rules always strip `;` (and it would have replaced that let with a const since it's not being reassigned)

The loop is unnecessary:  `const alreadyExist = this.props.flow.nodes.find(x => x.name === text)`

Reply:
Thanks for info, it seems `TSLint` doesn't work for `.jsx`, I will investigate this later.

Apologies for not being perfect-on-first-time, as I came from a different technology, always appreciate your support.

Reply:
That's my mistake, I forgot to mention that ESLint is also needed when playing with jsx files. 
 We will need to migrate rules from tslint to eslint in the future , since it's being deprecated.

Thank you very much for your contributions, it's really appreciated!